accesstoken/src/test/java/uk/gov/di/ipv/cri/common/api/handler/pact/utils/HandlerIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.di.ipv.cri.common.api.handler.pact.utils;
+
+import java.lang.reflect.Field;
+import java.net.HttpCookie;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public abstract class HandlerIntegrationTest<Q, S> {
+
+    protected Map<String, String> constructHeaders(Optional<HttpCookie> cookie) {
```

Comment:
`Optional` is for method return types and should ideally not be used in parameters.